components/qqr.js

```diff
@@ -0,0 +1,1198 @@
+// [qr.js](http://neocotic.com/qr.js)  
+// (c) 2014 Alasdair Mercer  
+// Licensed under the GPL Version 3 license.  
```

Comment:
Correct, would need to be a non gpl licensed version (e.g. bsd or mit licensed).